composer.json

```
@@ -16,20 +16,19 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
-        "mybuilder/cronos": "~3.0",
-        "doctrine/annotations": "1.*",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0",
-        "symfony/property-access": "^4.4 || ^5.0 || ^6.0",
-        "symfony/yaml": "^4.4 || ^5.0 || ^6.0"
+        "php": ">=8.0.2",
+        "mybuilder/cronos": "^3.0 || ^4.0",
+        "doctrine/annotations": "^1.14",
```

Thanks for looking at this 👍 
I've just tested and it seems to be ok from 1.7:
```suggestion
        "doctrine/annotations": "^1.7",
```

---

Sorry about that. I was on master and it was a different error I was getting 🤦 
1.11 is the one we want 👍

---

No problem. 1.11 worked